middleware/logger/logger.go

```diff
@@ -193,86 +193,39 @@ func New(config ...Config) fiber.Handler {
 			return nil
 		}
 
+		tagMap[TagTime] = func(buf *bytebufferpool.ByteBuffer, c *fiber.Ctx, w io.Writer, tag string) (int, error) {
```

Comment:
Each execution adds elements to the map, which is not necessary.

Reply:
Each execution adds elements to the map and It maybe very much affects

Reply:
fixed